gkeys/gkeys/action_map.py

```diff
@@ -385,16 +385,17 @@
         'func': 'updateseed',
```

Comment:
the commit message needs improvement.  It should state the improvements it added.  So, please change the summary line, then add several more lines stating the changes made.